linux/score-server/content/www/index.html

```diff
@@ -1,25 +1,59 @@
 <!doctype html>
 <html>
-  <head>
-    <title>Linux Workshop</title>
-    <meta charset="utf-8">
-    <meta http-equiv="refresh" content="10"> <!-- refresh page every N seconds -->
-    <style>
-     table {
-          width: 30%;
-          margin-left: auto;
-          margin-right: auto;
-      }
-      table, caption, th, td {
-        text-align: center;
-        border: 1px solid black;
-        border-collapse: collapse;
-        padding: 10px;
-      }
-    </style>
-  </head>
+
```

Comment:
I should be mad but this looks AMAZING

![image](https://github.com/opensourcecorp/workshops/assets/22351501/80e64c53-ffbb-42d9-a9c6-049227680696)

Reply:
But it *does* stretch across the given screen medium -- is there quick CSS voodoo we can do to pin its width to a proportion of the screen width?